src/frontend/src/layouts/PageTitle/PageTitle.tsx

```diff
@@ -7,7 +7,7 @@ import { Typography, Box, Grid, useTheme } from '@mui/material';
 import { ReactElement, ReactNode } from 'react';
 
 interface PageTitleProps {
-  title: string;
+  title: ReactNode;
```

Comment:
I'd like to avoid this if possible, it'd be cool if you can repurpose the chips prop to be more general, so it could accept smthn like an info bubble instead